alarms/alarm_manager.py

```diff
@@ -172,7 +172,14 @@ def trigger_pokemon(self, pkmn):
 			'dir': get_dir(lat,lng)
 		}
 		pkmn_info = self.optional_arguments(pkmn_info)
-			
+
+		#Check if the Pokemon will be back after a hidden phase
+		#Addinfo values: 1=2x15 point, 2=1x60h2 point, 3=1x60h3 point, 4=1x60h23 point, only needs to be added, if scanned before the break
+		addinfo = pkmn.get('addinfo',0)
```

Comment:
This all should be wrapped in a function in utils. Ideally we would just add one parameter into the pokemon info that calls this function to get the fields value.